src/test/java/org/fundacionjala/coding/rodrigo/movierefactor/MovieTest.java

```diff
@@ -0,0 +1,84 @@
+package org.fundacionjala.coding.rodrigo.movierefactor;
+
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+/**
+ * Test for the Movie's class.
+ */
+public class MovieTest {
+    private Movie movie1;
+    private Movie movie2;
+    private Movie movie3;
+
+    private Movie movie4;
+    private Movie movie5;
+    private Movie movie6;
+    private static final int FOUR = 4;
+    private static final int TEN = 10;
+    private static final int ONE = 1;
+
+    /**
+     * Setting the movies.
+     */
+    @Before
+    public void setUp() {
+
+        movie1 = new MovieRegular("The ducks", FOUR);
+        movie2 = new MovieChildrens("The papers", TEN);
+        movie3 = new MovieNewRelease("The FF", FOUR);
+        movie4 = new MovieRegular("The test", ONE);
+        movie5 = new MovieChildrens("The test revenge", ONE);
+        movie6 = new MovieRegular("The test coming soon", ONE);
```

Comment:
All these instances will be used in all the tests?
I don't think so
Please move them inside test method.